### PR TITLE
Implement form-help and form-error text components into form-control-text.tsx

### LINKED ISSUE #572: Implement form-help and form-error helper components to form-control-text
Building on work completed in #569

Let's implement `form-control-help-text` and `form-control-error-text` components into `form-control-text`.

### PR DESCRIPTION
**Pull request by colganmeanor**
### Implement `form-help-text` and `form-error-text` components to `form-control-text` inputs. 

~NOTE: This PR should be merged AFTER #570 is merged into main.~

- This PR adjusts the behavior of `form-control-text` by implementing the new help and error text components. The layout and styling of this component was slightly adjusted to make room for this - the component is now wrapped in a `div` with the bootstrap class `form-group` as well as some basic flex column properties. 
- Deleted a few scss files, and removed the shadow dom from some components that don't need them. Trying to eventually move to housing all form styles in the parent components. 
- I have changed the name of the `error` prop on this component to `errorText` to be more descriptive of the prop as it is passed to the error text component. The bulk of the files changed in this PR are related to the changing of this prop name. 
- Additionally, I deleted some older un-used component files that were created for the `owners` section of the original business form (we are not using these files, and will not use them in the future). 
- Lastly - updated test files in response to these changes. 

Links
-----

Closes #572 

Developer considerations
--------------

- On any task
  - [X] Previous unit and e2e tests are passing
  - [ ] Perform dev QA

Developer QA steps
--------------------

- [x] Test suites are passing 
- [x] Error validation text on form-control-text inputs should appear just as it did previously, but instead of `<div class='invalid-feedback' />` you should see the `form-control-error-text` component which renders a `<small class='text-danger' />` element instead
- [x] There is nowhere in the web component library where we are currently passing any help text to a `form-control-text` input, but try to add one to one of the fields in the business form and verify that it renders the `form-control-help-text` component, which should render `<small class='text-muted'>` element

### CODE REVIEW

File: packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.scss [hunk 1]
```diff
@@ -56,6 +56,13 @@
   }
 }
 
+input[type='text'][disabled] {
```
Comment (jakemer) [inline packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.scss:59]: I believe this should be handled by bootstrap - do we know why it's not?

Comment (colganmeanor) [reply packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.scss:59]: I'm not personally sure why this rule was written. It used to exist in the scss file for `form-control-text`, which I deleted the file and moved this rule to the top level scss file, `payment-provisioning.scss`

Here is the disabled text input with the bootstrap default styles (removing this rule declaration for `input`):

Screenshot 2024-06-27 at 2 42 41 PM

And, here is the disabled text input with the styles that someone wrote for this input, and the way it has looked for quite a while. 

Screenshot 2024-06-27 at 2 44 08 PM


I guess I'm fine either way, what do you think?

Comment (jakemer) [reply packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.scss:59]: I may have done that myself, but we should use the bootstrap default and remove instances of rolling our own disabled styles

**Review by jakemer — APPROVED** [review]

**Review by TiagoFuelber — APPROVED** [review]
Awesome! Approved!

**Review by TiagoFuelber — APPROVED** [review]
Awesome! Approved!

**Pull request merged**